server.go

```diff
@@ -155,6 +155,7 @@ func (s *Server) handleNextGame(rw http.ResponseWriter, req *http.Request) {
 	var request struct {
 		GameID  string   `json:"game_id"`
 		WordSet []string `json:"word_set"`
+		CreateNew bool   `json:"create_new"`
```

Comment:
```suggestion
		GameID    string   `json:"game_id"`
		WordSet   []string `json:"word_set"`
		CreateNew bool     `json:"create_new"`
```
nit: so far this project has been formatted using `gofmt`: https://blog.golang.org/gofmt